tools/mapping-scaling-tester/create_plots.py

```diff
@@ -191,34 +187,115 @@ def plotMapDataTime(df, prefix):
     ax.set_xlabel("edge length(h) of mesh A")
     ax.set_ylabel("time to map Data [ms]")
 
-    # plotConv(ax, df, yname)
-
     plt.gca().invert_xaxis()
     plt.grid()
     plt.savefig(prefix + "-mapt.pdf")
 
 
+def plot_scale_memory(df, prefix, participant):
```

Comment:
I could imagine that the plotting functions here can be further simplified by adding one 'low-level' and parametrized plotting function and calling this function for each variable?!